src/qibolab/serialize.py

```diff
@@ -48,10 +49,12 @@ def load_qubits(
         q: Qubit(q, **char)
         for q, char in runcard["characterization"]["single_qubit"].items()
     }
+
     if extras_folder is not None:
-        single_qubit = runcard["characterization"]["single_qubit"]
-        for qubit in qubits.values():
-            qubit.kernel_path = extras_folder / single_qubit[qubit.name]["kernel_path"]
+        kernels = Kernels.load(path=extras_folder / "kernels.npz")
```

Comment:
When using it I found here that I would like

```python
Kernels(kernels).dump(Path(__file__).parent / "dummy" / KERNELS_FILE)
```

to be something like 

```python
Kernels(kernels).dump(Path(__file__).parent / platform.name / KERNELS_FILE)
```

but for some reason we it can be named `"dummy"` or `"dummy_couplers"` should we join them ?